### PR TITLE
Fat Naga tail addition

### PR DESCRIPTION
**Pull request by lilBlovy**
## About The Pull Request

A ported over sprite from Vore Station

I added a new folder inside of the modular_zubbers Master folder that allows us to add more taurs in the future.

I have tested it and it works

## Why It's Good For The Game

It allows Nagas of all sizes to exist

Fat tail is Nice Tail. Very very nice :3

## Changelog

:cl:
add: Taurs_Types.dms folder
add: NagaFat Sprites
/:cl:

HumanBackview
HumanFrontview
HumanSideView
OnStationHumanBackView
OnStationHumanFrontView
OnStationHumanSideview

### CODE REVIEW

File: tgstation.dme [hunk 1]
```diff
@@ -8326,6 +8326,7 @@
 #include "modular_zubbers\modules\customization\modules\language\_language_holder.dm"
 #include "modular_zubbers\modules\customization\modules\language\nekomimetic.dm"
 #include "modular_zubbers\modules\customization\modules\language\piratespeak.dm"
+#include "modular_zubbers\modules\customization\modules\mob\dead\taur_fatnaga.dm"
```
Comment (aKromatopzia) [inline tgstation.dme:8329]: change this for "modular_zubbers/modules/customization/modules/mob/dead/taur_types.dm"

Comment (lilBlovy) [reply tgstation.dme:8329]: I accadently hit resolve so ignore that. How do I edit it within the browser.

**Comment by YakumoChen** [comment]
Needs pictures

**Comment by lilBlovy** [comment]
> Needs pictures

Will do.

**Comment by lilBlovy** [comment]
> ## About The Pull Request
> A ported over sprite from Vore Station
> 
> I added a new folder inside of the modular_zubbers Master folder that allows us to add more taurs in the future.
> 
> I have tested it and it works
> 
> ## Why It's Good For The Game
> It allows Nagas of all sizes to exist
> 
> Fat tail is Nice Tail. Very very nice :3
> 
> ## Changelog
> 🆑 add: Taurs_Types.dms folder add: NagaFat Sprites /:cl:

[FatNagaTail.zip](https://github.com/Bubberstation/Bubberstation/files/13767969/FatNagaTail.zip)

**Comment by lilBlovy** [comment]
> Needs pictures

It is done, check the file in the Comments

**Comment by YakumoChen** [comment]
That isn't a picture, that's a zip file
Post a screenshot from in game

**Comment by projectkepler-ru** [comment]
Can you include ingame screenshot?

**Comment by lilBlovy** [comment]
> That isn't a picture, that's a zip file Post a screenshot from in game
> 
> edit: that is what the zip file contains but you can (and should) just add them directly to the OP instead of zipping them

The Pictures are IN the ZIP file

**Comment by lilBlovy** [comment]
> Can you include ingame screenshot?

Look at the pictures in the zip file

**Comment by lilBlovy** [comment]
LoadoutBackView
LoadoutFrontView
LoadoutSideView
OnStationBackView
OnstationFrontview
OnStationSideView

**Comment by YakumoChen** [comment]
Could you post a screenshot of an actual human and not what looks like a slimeperson? It doesn't do the appearance justice at all, you should be presenting a baseline.

It overall doesn't look very good as it is. It's very blobby, but I'm not a spriter myself

**Comment by lilBlovy** [comment]
> Could you post a screenshot of an actual human and not what looks like a slimeperson? It doesn't do the appearance justice at all, you should be presenting a baseline.
> 
> It overall doesn't look very good as it is. It's very blobby, but I'm not a spriter myself

The point of it is for it to be blobby, but Yes I can send more pictures

**Comment by lilBlovy** [comment]
HumanBackview
HumanFrontview
HumanSideView
OnStationHumanBackView
OnStationHumanFrontView
OnStationHumanSideview

**Comment by Majkl-J** [comment]
Not reviewing this but personally don't like the sprite it feels like it doesn't fit in the current space.

**Comment by lilBlovy** [comment]
I would also like to share that I did show it to alot of people on the discord and they seemed to like it.

**Comment by lilBlovy** [comment]
I believe the photos dont serve it justice. When in motion it looks very nice and would be a great addition to the server. Its a port from Vorestation so I did not make it my self, but I do believe that once its in the game, It will look great and alot of people will love it.

**Comment by YakumoChen** [comment]
unfortunately 'just trust me bro' is not a great reason why it is good for the game

**Comment by lilBlovy** [comment]
It might not be for everyone, but there would probably be alot of people that do like it.

**Comment by lilBlovy** [comment]
> unfortunately 'just trust me bro' is not a great reason why it is good for the game
> 
> The sprite appears to have no animation so there will be no different when in motion or not.

And when I say "In motion" I mean when people are moving around with them.

**Comment by Majkl-J** [comment]
> I believe the photos dont serve it justice. When in motion it looks very nice and would be a great addition to the server. Its a port from Vorestation so I did not make it my self, but I do believe that once its in the game, It will look great and alot of people will love it.

Honestly, just open it up in anything, grab the colors around the edges, and make them ever so slightly more distinct. I think it will do wonders for it

**Comment by lilBlovy** [comment]
> > I believe the photos dont serve it justice. When in motion it looks very nice and would be a great addition to the server. Its a port from Vorestation so I did not make it my self, but I do believe that once its in the game, It will look great and alot of people will love it.
> 
> Honestly, just open it up in anything, grab the colors around the edges, and make them ever so slightly more distinct. I think it will do wonders for it

Let me see if I can do that.

**Comment by lilBlovy** [comment]
OnStationDarkenBackView
OnStationDarkenFrontView
OnStationDarkenSideView

**Comment by lilBlovy** [comment]
> OnStationDarkenBackView OnStationDarkenFrontView OnStationDarkenSideView

Updated with an darken outline, be sure to give your thoughts on it.

**Comment by Majkl-J** [comment]
I think that actually works quite nicely! I'm fine with the sprite now but spritework isn't shit I have much approve word over, so

**Pull request merged**